scripts/check-baseline.sh

```
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+echo "=== Baseline Check ==="
+echo ""
+
+echo "1. Running bertie tests..."
+cd /home/raditha/csi/Antikythera/bertie
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)